dev/gqltest/search_test.go

```diff
@@ -821,6 +827,11 @@ func TestSearch(t *testing.T) {
 						t.Fatal("Want non-zero results but got 0")
 					}
 				}
+				if test.exactMatchCount != 0 {
+					if results.MatchCount != test.exactMatchCount {
+						t.Fatalf("Want exactly %d results but got %d", test.exactMatchCount, results.MatchCount)
+					}
+				}
```

Comment:
nit: since we don't have an else branch:
```suggestion
				if test.exactMatchCount != 0 && results.MatchCount != test.exactMatchCount {
					t.Fatalf("Want exactly %d results but got %d", test.exactMatchCount, results.MatchCount)
				}
```